lib/rules/implicit-arrow-linebreak.js

```diff
@@ -37,6 +43,130 @@ module.exports = {
             return context.options[0] || "beside";
         }
 
+        /**
+         * Formats the comments depending on whether it's a line or block comment.
+         * @param {Comment[]} comments The array of comments between the arrow and body
+         * @param {Integer} column The column number of the first token
+         * @returns {string} A string of comment text joined by line breaks
+         */
+        function formatComments(comments, column) {
+            const whiteSpaces = " ".repeat(column);
+
+            return `${comments.map(comment => {
+
+                if (comment.type === "Line") {
+                    return `//${comment.value}`;
+                }
+
+                return `/*${comment.value}*/`;
+            }).join(`\n${whiteSpaces}`)}\n${whiteSpaces}`;
+        }
+
+        /**
+         * Finds the first token to prepend comments to depending on the parent type
+         * @param {Node} node The validated node
+         * @returns {Token|Node} The node to prepend comments to
+         */
+        function findFirstToken(node) {
+            switch (node.parent.type) {
+                case "VariableDeclarator":
+
+                    // find the grandparent, i.e. const, let, var
+                    return sourceCode.getFirstToken(node.parent.parent);
```

Comment:
What will happen here if the parent `VariableDeclaration` has multiple `VariableDeclarator` children? For example:

```js
const foo = 1,
    bar = 2,
    baz = () => //comment
        qux
```

Will that result in the comment getting hoisted to the top of the statement, before `const`?

Reply:
HI @not-an-aardvark, I added a couple test cases for when the parent `VariableDeclaration` has multiple `VariableDeclarator` children -- the comment should get hoisted to the top of the statement if the node is the first/only child, otherwise, the declarator itself.